core/src/main/java/org/mapfish/print/VersionInfo.java

```diff
@@ -64,7 +66,8 @@ private String getValue(final String name) {
     }
 
     private Attributes getAttributes() {
-        if (this.servletContext == null) {
+        if (this.servletContext == null ||
+                this.servletContext.getClass().getSimpleName().startsWith("Mock")) {
```

Comment:
same above

Reply:
See point 4

Reply:
When generating the javadoc, I was annoyed by a huge stacktrace. I hate stacktraces when I'm generating javadoc. ;-)

Reply:
Relay nice, many thanks :-)